operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeploymentDependentResource.java

```diff
@@ -419,7 +422,14 @@ private List<EnvVar> getDefaultAndAdditionalEnvVars(Keycloak keycloakCR, String
                     }
                     return envBuilder.build();
                 })
-                .collect(Collectors.toList());
+                .collect(Collectors.toCollection(ArrayList::new));
+
+        for (String env : COPY_ENV) {
+            String value = System.getenv(env);
+            if (value != null) {
+                envVars.add(new EnvVar(env, value, null));
```

Comment:
```suggestion
                envVars.add(new EnvVarBuilder().withName(env).withValue(value).build());
```

nitpick: I'd prefer to use the builder for it, as I can imagine some default `EnvVarSource` in the future. But it is only a nitpick, not a big deal.